media_driver/linux/common/os/i915/mos_bufmgr.c

```diff
@@ -1611,25 +1756,27 @@ mos_gem_bo_map_wc(struct mos_linux_bo *bo) {
         return ret;
     }
 
-    /* Now move it to the GTT domain so that the GPU and CPU
-     * caches are flushed and the GPU isn't actively using the
-     * buffer.
-     *
-     * The domain change is done even for the objects which
-     * are not bounded. For them first the pages are acquired,
-     * before the domain change.
-     */
-    memclear(set_domain);
-    set_domain.handle = bo_gem->gem_handle;
-    set_domain.read_domains = I915_GEM_DOMAIN_GTT;
-    set_domain.write_domain = I915_GEM_DOMAIN_GTT;
-    ret = drmIoctl(bufmgr_gem->fd,
+    if (!bufmgr_gem->has_lmem) {
```

Comment:
Refer to https://github.com/freedesktop/drm-tip/commit/81340cf3bddde, for discrete GPU, we do not need flush the cache anymore. But we still need to wait the bo ready before the CPU can access the buffer.

Reply:
done